scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/entrypoints/AutoBattle.kt

```diff
@@ -460,7 +461,10 @@ open class AutoBattle @Inject constructor(
             3.seconds.wait()
         } else if (waitAPRegenPrefs) {
             Location(1300, 1240).click();
-            20.seconds.wait()
+            for (i in 1..waitAPRegenMinutePrefs) {
+                toast("Waiting for AP regeneration. Re-checking in "+ (waitAPRegenMinutePrefs-i+1) + " minutes.")
```

Comment:
```suggestion
                toast("Waiting for AP regeneration. Re-checking in ${waitAPRegenMinutePrefs - i + 1} minutes.")
```

just a Kotlin feature